pkg/channel/multichannelfanout/multi_channel_fanout_handler_test.go

```diff
@@ -17,12 +17,13 @@ limitations under the License.
 package multichannelfanout
 
 import (
-	"fmt"
+	"context"
 	"net/http"
 	"net/http/httptest"
-	"strings"
 	"testing"
 
+	"github.com/cloudevents/sdk-go"
```

Comment:
I think ther are some more sections w/o the alias

Let's be consistent, and always use it (fine to have that in a separate PR to minimize the "noise"?), @nachocano

Reply:
let me do a find and add it everywhere... thanks for the catch..

Reply:
Done!!! thanks @matzew